packages/gp2-components/src/molecules/IconWithLabel.tsx

```diff
@@ -0,0 +1,25 @@
+import { pixels } from '@asap-hub/react-components';
+import { css } from '@emotion/react';
+
+const { rem } = pixels;
+
+type IconWithLabelProps = {
+  readonly icon: JSX.Element;
+};
+const styles = css({
+  display: 'flex',
+  gap: rem(9),
+  margin: `${rem(8)} 0`,
+});
+const iconStyles = css({
+  display: 'inline-flex',
+});
+
+const IconWithLabel: React.FC<IconWithLabelProps> = ({ icon, children }) => (
```

Comment:
Not sure if it's important to formattiing but `const`s are separated by an empty line here but not in the above.

Reply:
I don't think we have a rule for it